docs/Demo_Projects/Edge_Impulse_demo.md

```diff
@@ -207,4 +207,4 @@ Now start moving the Thingy:91 around to detect all three patterns. When a new m
 
 The LwM2M standard allows you to instruct the device about the minimum and maximum allowed intervals between two updates. The intervals can be set from 1 second to multiple days. To instruct the device, an OBSERVE operation can be sent from the LwM2M Server to the LwM2M Client.
 
-For more information, see [how to set observations](/Coiote_IoT_DM/User_Interface/Device_Center/#set-observation)
\ No newline at end of file
+For more information, see [how to set observations](/Coiote_IoT_DM/Device_Center/#set-observation)
```

Comment:
works fine, but the plugin I use for documentation link validation returns an error:
the version below should be more more versatile I guess (+full stop at the end)

`For more information, see [how to set observations](../Coiote_IoT_DM/Device_Center.md#set-observation).`

Reply:
Thanks for the suggestion.

To make this work, it should be: `../../Coiote_IoT_DM/Device_Center.md#set-observation`

When using `/` it directly starts at the root folder (which I personally prefer).